build.cmd

```diff
@@ -19,16 +19,18 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 rem copy nuget locally
-if not exist .nuget\NuGet.exe (
-  if not exist .nuget md .nuget
-  copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
-)
+if not exist .nuget md .nuget
+copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
 
 rem restore packages for build script
 .nuget\NuGet.exe restore .\packages.config -PackagesDirectory .\packages -Verbosity quiet
 
+rem find Visual Studio and MSBuild installation paths
+FOR /F "delims=" %%i IN ('.\packages\vswhere.1.0.62\tools\vswhere.exe -property installationPath') DO set VS_PATH=%%i

```

Comment:
I was thinking more of network than storage, so good point about the 6.58.

It still feels a little odd to add the package dependency when we _know_ there's a Roslyn just sitting on the disk. Sort of like running GitVersion twice, when we could've just cached variables from the first run. :wink:

Okay. If you want to leave this and the nuget copy as is, I can merge after a squash. Or I'm happy to wait for any change.